tests/kafkatest/services/verifiable_producer.py

```diff
@@ -63,6 +65,11 @@ def start_cmd(self):
             cmd += " --max-messages %s" % str(self.max_messages)
         if self.throughput > 0:
             cmd += " --throughput %s" % str(self.throughput)
+        cmd += " --close-timeout %s" % str(self.close_timeout_sec)
+
+        if self.producer_props is not None:
```

Comment:
Could change the default to be an empty list (or zero-tuple if you prefer making it immutable), which avoids `is not None` checks like this littering the code.